gradle.properties

```diff
@@ -1,3 +1,3 @@
-org.gradle.parallel=true
-org.gradle.configureondemand=true
-org.gradle.cache.tasks=true
+org.gradle.parallel=false
+org.gradle.configureondemand=false
+org.gradle.cache.tasks=false
```

Comment:
I saw, thats fine. `org.gradle.configureondemand=true` can speed up the build a bit but its not really that important in this build because its so small.